packages/components/src/components/as-category-widget/README.md

```diff
@@ -240,13 +240,23 @@ categoryWidget.addEventListener('categoriesSelected', event => {
 
 #### **getSelectedCategories**
 Get current selected categories
-`Returns: Category[]`
+`Returns: Promise<Category[]>`
 
 ```code
 lang: javascript
 ---
+// Async/Await approach
+// You need to wrap your code in an async function in top level scripts
+(async function () {
+    const categoryWidget = document.querySelector('as-category-widget');
+    await categoryWidget.getSelectedCategories();
+})();
+
```

Comment:
I'd put the simple code (no IIFE) and then a note telling that if you need to use it in a top level script you need to wrap, with a link to a deeper explanation.